.github/workflows/release.yaml

```diff
@@ -30,14 +32,50 @@ jobs:
         run: make GIT_TAG=${{ github.event.inputs.tag }} -f builder.Makefile cross
 
       - name: Compute checksums
-        run: cd bin; for f in *; do shasum --binary --algorithm 256 $f | tee -a checksums.txt > $f.sha256; done
+        working-directory: bin/
```

Comment:
note: this change is not needed. I added `working-directory` to avoid the `cd bin`, but I can revert.